addons/hr/static/src/js/chat_mixin.js

```diff
@@ -23,11 +23,11 @@ const ChatMixin = {
         return this._super();
     },
 
-    _onOpenChat: function (ev) {
+    async _onOpenChat(ev) {
         ev.preventDefault();
         ev.stopImmediatePropagation();
-        const env = Component.env;
-        env.messaging.openChat({ employeeId: this.state.data.id });
+        const messaging = await Component.env.services.messaging.modelManager.getMessaging();
```

Comment:
Could avoid leaking implementation detail of model with nicer use from the messaging service, like:
`await env.services.messaging.get()`.